task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```
@@ -649,20 +662,21 @@ spec:
 
         buildah commit "${BUILDAH_ARGS[@]}" $container $IMAGE
 
-        status=-1
-        max_run=5
-        sleep_sec=10
-        for run in $(seq 1 $max_run); do
-          status=0
-          [ "$run" -gt 1 ] && sleep $sleep_sec
-          echo "Pushing sbom image to registry"
-          buildah push \
-            --tls-verify=$TLSVERIFY \
-            --digestfile /var/workdir/image-digest $IMAGE \
-            docker://$IMAGE && break || status=$?
-        done
-        if [ "$status" -ne 0 ]; then
-          echo "Failed to push sbom image to registry after ${max_run} tries"
+        echo "Pushing to ${IMAGE%:*}:${TASKRUN_NAME}"
+        if ! retry buildah push \
+          --tls-verify="$TLSVERIFY" \
+          "$IMAGE" \
+          "docker://${IMAGE%:*}:$(context.taskRun.name)"; then
+          echo "Failed to push sbom image to ${IMAGE%:*}:$(context.taskRun.name) after ${max_run} tries"
+          exit 1
+        fi
```

IMO this makes readability better; but just nitpick and personal preference
```suggestion
        echo "Pushing to ${IMAGE%:*}:${TASKRUN_NAME}"
        if ! retry buildah push \
          --tls-verify="$TLSVERIFY" \
          "$IMAGE" \
          "docker://${IMAGE%:*}:$(context.taskRun.name)";
         then
          echo "Failed to push sbom image to ${IMAGE%:*}:$(context.taskRun.name) after ${max_run} tries"
          exit 1
        fi
```

---

I meant:
```
if [cond]
then  # on newline
  code
fi
```

---

work for me